chart/templates/dag-processor/dag-processor-deployment.yaml

```
@@ -227,7 +227,7 @@ spec:
             {{- end }}
           {{- end }}
         {{- if .Values.dagProcessor.extraContainers }}
-          {{- toYaml .Values.dagProcessor.extraContainers | nindent 8 }}
+          {{- tpl (toYaml .Values.dagProcessor.extraContainers) . | nindent 8 }}
```

Don't you need to template first and then convert to YAML?

```suggestion
          {{- toYaml (tpl .Values.dagProcessor.extraContainers) . | nindent 8 }}
```

---

I understand what u saying and it sound correct but when swapping it I get an error.
Also when looking at the chart the order is like I did and not like u suggest.